chord_metadata_service/mohpackets/apis/discovery.py

```diff
@@ -104,23 +103,8 @@ def discover_sidebar_list(request):
     Retrieve the list of drug names and treatment for frontend usage
     """
     # Drugs queryable for chemotherapy
-    chemotherapy_drug_names = list(
-        Chemotherapy.objects.exclude(drug_name__isnull=True)
-        .values_list("drug_name", flat=True)
-        .order_by("drug_name")
-        .distinct()
-    )
-    # Drugs queryable for immunotherapy
-    immunotherapy_drug_names = list(
-        Immunotherapy.objects.exclude(drug_name__isnull=True)
-        .values_list("drug_name", flat=True)
-        .order_by("drug_name")
-        .distinct()
-    )
-
-    # Drugs queryable for hormone therapy
-    hormone_therapy_drug_names = list(
-        HormoneTherapy.objects.exclude(drug_name__isnull=True)
+    systemictherapy_drug_names = list(
```

Comment:
For readability, should be systemic_therapy
```suggestion
    systemic_therapy_drug_names = list(
```